src/components/Comments/Comment.js

```diff
@@ -273,11 +338,18 @@ const Comment = ({
           </ButtonHightlight>
         </TouchableOpacity>
 
-        {totalVote != 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+        <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
         <TouchableOpacity activeOpacity={1} testID="upvoteBtn">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for displaying the total vote count has been removed, meaning the vote count will always be displayed even if it's zero. Ensure this is the intended behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->